docs/what.rst

```diff
@@ -0,0 +1,29 @@
+=====================
+What is a Certificate?
+=====================
+
+A public key or digital *certificate* (formerly called an SSL certificate) uses a public key 
+and a private key to enable secure communication between a client program (web browser, email client, 
+etc.) and a server over an encrypted SSL (secure socket layer) or TLS (transport layer security) connection.
+The certificate is used both to encrypt the communication and to identify the server. The certificate
+includes information about the key, information about the server identity, and the digital signature
+of the certificate issuer. If the issuer is trusted by the software that initiates the communication,
+and the signature is valid, then the key can be used to communicate securely with the server identified by 
+the certificate. Using a certificate is a good way to prevent "man-in-the-middle" attacks, in which
+someone in between you and the server you think you are talking to is able to insert their own (harmful)
+content.
+
+You can use Certbot to easily obtain and configure a free certificate from Let's Encrypt, a
+joint project of EFF, Mozilla, and many other sponsors.
+
+Certificates and Lineages
+=========================
+
+Certbot introduces the concept of a *lineage,* which is a collection of all the versions of a certificate
+plus additional information that Certbot uses to maintain the configuration of your webserver from
```

Comment:
Should it be just "server" instead of "webserver"?
In principle, one can have certbot running on a server without any webserver, right?

Thinking more about that, I started questioning: is it really the configuration of the server that is saved, or rather the configuration of _certbot_? Yes, it may _reflect_ the server configuration, but only in the part (amount) and the way that is used for configuring _certbot_. 
Moreover, it is actually a configuration pertaining only to the particular certificate. (And in the next sentence, you are actually referring to this _Certbot ... configuration_
So, I am proposing "to maintain the configuration of certbot for that certificate".

Furthermore, we can simplify that to:
```
-plus additional information that Certbot uses to maintain the configuration of your webserver from
+plus configuration of certbot for that certificate maintained from
```